src/env/builtins/lredir.c

```diff
@@ -543,6 +482,12 @@ int lredir_main(int argc, char **argv)
       return(0);
     }
 
+    /* check the MFS redirector supports this DOS */
+    if (!isInitialisedMFS()) {
+      printf("Unsupported DOS version\n");
```

Comment:
> Since we still support the behind scenes initialisation of the redirector we don't rely
> upon emufs.sys being loaded.

Yes, but this doesn't mean in case of a failed init
(including the old fall-back init) we can't pretend
that emufs.sys is needed. :) I think this would be
nice for the future.

> I can add a log message to mfs.c, but would you prefer on init, on status query, or both?

I think it would be better to add it somewhere to
the caller, maybe mfs.sys should only return the
detailed status. Then also emufs.sys can print
such error.